src/components/MapPage/index.js

```diff
@@ -41,6 +58,22 @@ const MapPage = () => {
         </form>
       }
       {userLocation.length > 0 && <Atlas userLocation={userLocation} />}
+      <p>Center is {closestLocation.distance} miles away</p>
+      <section> 
+        <h3>GET DIRECTIONS</h3>
+        <a href={drivingUrl} target="_blank" rel="noopener noreferrer">
+          <img src={require("../../assets/images/icons/car.svg")} alt="car directions link"/>
+        </a>
```

Comment:
Oh! You import images inline!! I didn't even know that was a thing you could do! I always end up with a billion imports at the top when I use a lot of icons and things.